x/feesplit/types/msg.go

```diff
@@ -50,10 +55,9 @@ func (msg MsgRegisterFee) ValidateBasic() error {
 		return sdkerrors.Wrapf(err, "invalid contract address %s", msg.ContractAddress)
 	}
 
-	// WithdrawAddress can be omitted and it will default to DeployerAddress
-	if msg.WithdrawAddress != "" {
-		if _, err := sdk.AccAddressFromBech32(msg.WithdrawAddress); err != nil {
-			return sdkerrors.Wrapf(err, "invalid withdraw address address %s", msg.WithdrawAddress)
+	if msg.WithdrawerAddress != "" {
+		if _, err := sdk.AccAddressFromBech32(msg.WithdrawerAddress); err != nil {
+			return sdkerrors.Wrapf(err, "invalid withdraw address %s", msg.WithdrawerAddress)
```

Comment:
Nit: A little typo in `invalid withdraw address %s` according to the new changes, it should be `withdrawer`, right?